spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.feature 'Post Index', type: :feature do
```

Comment:
- [ ] I suggest that you add a test that asserts you can see a section for pagination if there are more posts than fit on the view as required in the project requirement :blush:

Reply:
- [ ] [REQUIRED] As stated by the previous reviewer, kindly add a test that asserts you can see a section for pagination if there are more posts than fit on the view as required in the project requirement.

Reply:
I'm Confused now 
Which one is it Optional or Required
I was going to come back and fix it even if it is optional
If it is required why not just say so
could have saved me a day